app/assets/javascripts/scp-igv.js

```diff
@@ -35,15 +35,19 @@ $(document).on('click', '.bam-browse-genome', function(e) {
     selectedBams[thisBam] = 1;
   }
 
-  $('#genome-tab-nav').css('display', ''); // Show 'Genome' tab
   $('#study-visualize-nav > a').click();
   $('#genome-tab-nav > a').click();
-
-  hasDisplayedIgv = true;
 });
 
 $(document).on('click', '#genome-tab-nav', function (e) {
-  initializeIgv();
+  window.location.hash = '#genome-tab';
+
+  // Go to Google sign-in page, redirect to this view after authenticating.
+  if (accessToken === null) {
+    ga('send', 'event', 'genome', 'sign_in_start');
+    localStorage.setItem('signed-in-via-genome-tab', true);
+    window.location = '/single_cell/users/auth/google_oauth2';
```

Comment:
This isn't something that needs to be change, but since we can't interpolate Ruby in here since this is pure javascript, this is something that we'll have to make a note of if we ever change our URL.

Reply:
Good point, agreed.